sys/src/9/amd64/syscall.c

```diff
@@ -532,21 +532,14 @@ sysprocsetup(Proc* p)
 void
 sysrforkchild(Proc* child, Proc* parent)
 {
-	Ureg *cureg;
-// If STACKPAD is 1 things go very bad very quickly.
-// But it is the right value ...
-#define STACKPAD 1 /* for return PC? */
-	/*
-	 * Add STACKPAD*BY2SE to the stack to account for
-	 *  - the return PC
-	 *  (NOT NOW) - trap's arguments (syscallnr, ureg)
-	 */
-	child->sched.sp = PTR2UINT(child->kstack+KSTACK-((sizeof(Ureg)+STACKPAD*BY2SE)));
-	child->sched.pc = PTR2UINT(sysrforkret);
+	char *cureg;
```

Comment:
should this be a separate PR; I'd like to know why you were able to remove STACKPAD.

Reply:
Sure; reverted for now.